SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -201,48 +229,77 @@ void USpatialNetDriver::InitiateConnectionToSpatialOS(const FURL& URL)
 
 	if (!bPersistSpatialConnection)
 	{
-		// Destroy the old connection
 		GameInstance->DestroySpatialWorkerConnection();
-
-		// Create a new SpatialWorkerConnection in the SpatialGameInstance.
 		GameInstance->CreateNewSpatialWorkerConnection();
 	}
 
 	Connection = GameInstance->GetSpatialWorkerConnection();
 
-	if (URL.HasOption(TEXT("locator")))
+	bool bShouldLoadFromURL = true;
+	bool bUseReceptionist = true;
+
+	// If this is the first connection try using the command line arguments to setup the config objects.
+	// If arguments can not be found we will use the regular flow of loading from the input URL.
+	if (!GameInstance->bFirstConnectionToSpatialOSAttempted || URL.Host == SpatialConstants::RECONNECT_USING_ARGUMENTS)
```

Comment:
So, in the case I have a game which starts with the args "MyGame.exe 175.3.81.1 +receptionistHost=127.0.0.1 +receptionistPort=1.1.1.1" 
The initial connection will attempt to use receptionist with 127.0.0.1?

Reply:
I think a chonky comment block and docs update might be in order.

Reply:
There is a separate PR for the documentation.